backend/utils/oauth_utils.py

```diff
@@ -36,9 +36,16 @@ def get_or_create_user(user_info):
     username = user_info["login"]
     email = user_info["email"]
 
+    filterUser = User.objects.filter(email=email).first()
+    if not filterUser.id42:
```

Comment:
Cuidado con esto, que si filterUser is None... peta

Reply:
![imagen_2024-10-02_231041205](https://github.com/user-attachments/assets/a58443b1-5502-4fda-9a8b-6189a741ba5c)
tiene sentido esta intentado acceder aun atributo sin saber ni si existe el objeto